### PR TITLE
Day 4

### PR DESCRIPTION
**Pull request by qwibbler**
Add tables `visits` and `specializations` and queried them with 1M, M1 and MM relations.

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -72,4 +72,43 @@ SET owner_id = 4
 WHERE name in ('Charmander', 'Squirtle', 'Blossom');
 UPDATE animals
 SET owner_id = 5
-WHERE name in ('Angemon', 'Boarmon');
\ No newline at end of file
+WHERE name in ('Angemon', 'Boarmon');
+INSERT INTO vets (name, age, date_of_graduation)
+VALUES ('William Tatcher', 45, '2000-04-23'),
+  ('Maisy Smith', 26, '2019-01-17'),
+  ('Stephanie Mendez', 64, '1981-05-04'),
```
Comment (mmsesay) [inline data.sql:79]: - [ ] I think you made a mistake in vet `Stephanie Mendez` date of graduation. It should be `1981-05-17`. Please try to update it.

**Review by mmsesay — CHANGES_REQUESTED** [review]
Hi @qwibbler, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (mmsesay) [reply data.sql:79]: Hello, I made a required change and later noticed that you actually used the date given to you in your project requirements.
Apologies for that, it was a typo made in both review requirements and your project requirements.

Please ask for a re-review.
Thanks for understanding.

Comment (qwibbler) [reply data.sql:79]: No problem, thanks for the review!

**Review by AshakaE — APPROVED** [review]
### Hi @qwibbler   🙋‍♀️, 

# STATUS APPROVED ✔
   
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**